packages/plugin-ext/src/plugin/type-converters.spec.ts

```diff
@@ -287,6 +300,14 @@ describe('Type converters:', () => {
             assert.deepStrictEqual(result, shellTaskDto);
         });
 
+        it('should convert to task dto with ShellStringQuoted', () => {
+            const result: TaskDto | undefined = Converter.fromTask(shellPluginTaskWithQuotedCommand);
+
+            // then
+            assert.notStrictEqual(result, undefined);
+            assert.deepStrictEqual(result, shellTaskDto);
+        });
```

Comment:
minor: I believe this is still unaddressed from the previous review (https://github.com/eclipse-theia/theia/pull/10997#discussion_r848420705)

```suggestion
        it('should convert to task dto with ShellStringQuoted', () => {
            const result: TaskDto | undefined = Converter.fromTask(shellPluginTaskWithQuotedCommand);
            assert.notStrictEqual(result, undefined);
            assert.deepStrictEqual(result, shellTaskDto);
        });
```

Reply:
Ah... yes. You are right. Just pushed the fix.
Thanks for the quick response.